examples/example1.tfnot

```diff
@@ -0,0 +1,11 @@
+module "new_record_and_certificate" {
+  source = "github.com/StratusGrid/terraform-aws-acm-certificate-creation"
+
+  for_each        = toset(var.acm_certificate_name)
```

Comment:
Why would this need to be a set? And since it's a set we should provide an example of how the data should be inputed since it's a bit vague to me.

Reply:
Below is the error one will get if "toset" was removed and the acm_certificate_name list contained one value in it.
` The given "for_each" argument value is unsuitable: the "for_each" argument must be a map, or set of strings, and you have provided a value of type list of string.`

Reply:
That's fine, though can we provide an input of what it should look like?

For example

```
variable "acm_certificate_name"
description "Cert FQDN"
type = string
default = "test.example.com"
```